src/main/java/AlexLink/HomeWork/Exception/Car.java

```diff
@@ -0,0 +1,49 @@
+package AlexLink.HomeWork.Exception;
+
+public class Car {
+    private String brand;
+    private int speed;
+    private double price;
+
+    public Car() {
+    }
+
+    public Car(String brand, int speed, double price) {
+        this.brand = brand;
+        this.speed = speed;
+        this.price = price;
+    }
+
+    public String getBrand() {
+        return brand;
+    }
+
+    public void setBrand(String brand) {
+        this.brand = brand;
+    }
+
+    public int getSpeed() {
+        return speed;
+    }
+
+    public void setSpeed(int speed) {
+        this.speed = speed;
+    }
+
+    public double getPrice() {
+        return price;
+    }
+
+    public void setPrice(double price) {
+        this.price = price;
+    }
+
+    void start() throws StartException {
```

Comment:
you special use default access modifier or missed it?

Reply:
fixed it